packages/stryker/src/utils/Timer.ts

```diff
@@ -1,13 +1,17 @@
 
 export default class Timer {
   private start: Date;
+  private markers: {
+    [name: string]: Date;
+  };
 
-  constructor() {
+  constructor(private now = () => new Date()) {
```

Comment:
I don't really like the way this constructor is bloated:
1. It defines a variable `now`
2. It defaults to a lambda function that is defined inside the constructor body.

It would be nice to clean this one up.